static/js/alert.js

```diff
@@ -353,6 +353,7 @@ function displayAlertProperties(res) {
     const queryParams = res.queryParams;
     $('.alert-name').text(alertInfo.alert_name);
     $('.alert-status').text(mapIndexToAlertState.get(alertInfo.state));
+    $('.alert-status').text(alertStatus).css('background-color', backgroundColor);
```

Comment:
add clss to alert-status based on alertInfo.state and then use css to set the color.